doc/rules/basic/single_line_empty_body.rst

```diff
@@ -22,3 +22,11 @@ Example #1
    -{
    -}
    +) {}
+
+Rule sets
+---------
+
+The rule is part of the following rule set:
+
+- `@PhpCsFixer <./../../ruleSets/PhpCsFixer.rst>`_
```

Comment:
I will merge as-is, but it should rather became part of PER ruleset, and PHPCsFixer ruleset should reuse it
ref https://www.php-fig.org/per/coding-style/

Reply:
OK, I believe we will refactor it when finally someone gets hands on PER-CS 🙂.